src/objects/sap/zif_abapgit_sap_report.intf.abap

```diff
@@ -0,0 +1,54 @@
+INTERFACE zif_abapgit_sap_report
+  PUBLIC.
+
+  TYPES:
+    ty_abap_language_version TYPE c LENGTH 1.
+
+  METHODS read_report
+    IMPORTING
+      iv_name          TYPE programm
```

Comment:
for the `PROGRAMM`, can https://abapedia.org/steampunk-2305-api/syrepid.dtel.html be used instead?

Reply:
done